ruby-sdk/Cargo.lock

```diff
@@ -296,6 +296,8 @@ dependencies = [
 [[package]]
 name = "eppo_core"
 version = "1.0.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "d21810a56968888dc45c5b10df36fbff2cdc8e56881766d48a28ad395c3de3e6"
```

Comment:
This is the fix for the issue. The rest is to ensure we catch this error automatically in the future.